### ISSUE TITLE
Picker window, Subtitles, Encoding with Force On when shouldn't

### ISSUE THREAD

**Issue by DarkWolfNine** [RandomEngy/VidCoder#269]
![vidcoder_subs_picker](https://user-images.githubusercontent.com/30877277/32642101-d298b950-c586-11e7-9dd3-81b58f1a1611.jpg)
![vidcoder_subs_shouldnthaveforceon](https://user-images.githubusercontent.com/30877277/32642102-d2f7c63e-c586-11e7-8a27-acf19b2dc049.jpg)

The picker screenshot shows that I do have default on for the first subtitle.  But I don't have Force On checked..

The Media Info is showing that it is forcing the sub as well.

If I use MKVMerge with a similar (telling it to default the first, but don't force it) it doesn't turn Force On. But Default is enabled.

I hope I explained this well enough. This is with current VidCoder 2.59 (Windows installed)

**Comment by RandomEngy** [comment]
The "Forced only" checkbox doesn't control the output, it controls which subtitles are chosen from the input track. Some subtitles on the input track can be marked as "forced" and if you choose "forced only" it will only pick the ones with that special marking.

**Comment by DarkWolfNine** [comment]
Thank you for the explanation.
So what would set the Forced on the output? I tried encoding the same episodes before I found the handy picker menu and I did check the boxes on the main screen to include both subtitles, but I didn't check either default or forced on either of the two. The output included the 2 sub tracks, but both were default No and Forced No. - Apologies for the noobish question. I just found out about your app and I'm still learning it. Thank you so much, by the way. I'm finding it nicer than straight handbrake.
